hardware/is_camera/src/camera.cc

```diff
@@ -243,6 +245,25 @@ namespace is_camera {
     thread_ = std::thread(&CameraNodelet::PublishLoop, this);
   }
 
+  // Set the exposure
+  bool CameraNodelet::SetExposure(ff_msgs::SetExposure::Request  &req,
+                          ff_msgs::SetExposure::Response &res) {
```

Comment:
I thought I remembered working on something like this before... here it is https://github.com/nasa/astrobee/pull/689 . That PR was rejected because it caused some crashes but we never learned why.

Some things I like about that PR that could make it into this if it makes sense:
- It separates out `SetExposure()` and `SetGain()` functions so they can be called from wherever in a way that would make this PR more convenient. There is also a guard to avoid making a redundant syscall if the desired value is already the current value.
- I noticed some `ioctl()` calls that I guess should be converted to `xioctl()` in general.

I don't feel strongly about including this stuff if it feels too tangential, but it could be nice to get it in if it can piggyback on your testing plans.

Reply:
Hi Trey! Since I'm in a rush to merge it such that Katie can add the command to the executive before leaving, I'll delay doing this, but I'm always available for lab testing on any PR :)